src/Reducer/authReducer.js

```diff
@@ -0,0 +1,3 @@
+export const authReducerFtn = (state,action) => {
+    
+}
```

Comment:
If this file not needed you can remove this

Reply:
done

Reply:
done